### ISSUE TITLE
Current Build Breaks Layout on Chrome Browser

### ISSUE THREAD

**Issue by NotTheHerbie** [rstrouse/nodejs-poolController-dashPanel#31]
Yesterday I pulled the latest dashPanel commit and discovered the layout on Chrome appears to be broken.  The layout on Edge appears fine.  Both browsers are up-to-date.  I've included screen shots of both.  Prior to updating my local repo, the Chrome layout was similar to Edge's.
Chrome:
![image](https://user-images.githubusercontent.com/73130412/120898390-88e4b180-c5df-11eb-80bb-ac7587f6294f.png)

Edge:
![image](https://user-images.githubusercontent.com/73130412/120898419-a154cc00-c5df-11eb-806f-c823500096ac.png)

Let me know if you need any additional information.
Thanks,
Mike

**Comment by tagyoureit** [comment]
Did you do a hard refresh (ctrl-f5) on chrome?  It doesn't look like that for me.

**Comment by NotTheHerbie** [comment]
I had opened chrome in a new window, but I didn't do a hard refresh.  Today when I opened it in a new window, it was fine.  Sorry for the false alarm.

Thanks to you, rstrouse and all the other contributors to nodejs-poolController, dashPanel and relayEquipmentManager for a great open source alternative to Pentair's less than perfect offering.  

Now, all I need to do is come up to speed on InfluxDB and Grafana and I won't need to rely on Pentair's servers for my IntelliChem's pool chemistry history and IntelliChlor/MA usage.  Thanks again!
